jobs/scanning/covscan-images/Jenkinsfile

```diff
@@ -137,12 +137,15 @@ enabled_metadata=1
                         --local-repo-rhel-7 covscan-rhel-7_repos/covscan-testing
                         --local-repo-rhel-8 covscan-rhel-8_repos/covscan
                         --local-repo-rhel-8 covscan-rhel-8_repos/covscan-testing
+                        --local-repo-rhel-9 covscan-rhel-8_repos/covscan
+                        --local-repo-rhel-9 covscan-rhel-8_repos/covscan-testing
```

Comment:
This needs to point to rhel 9 repos. We need to create local repo mirrors at line 97 first.